recipe/meta.yaml

```diff
@@ -2,20 +2,26 @@ package:
     name: mingwpy
     version: 0.1.0
 
-source:                                                                                                # [win]
+source:
```

Comment:
Maybe we should fix the script instead of adding that to all feedstock that might skip stuff and create an empty field...